package.json

```diff
@@ -32,7 +32,8 @@
     "quality:format": "lerna run --parallel quality:format",
     "quality:lint": "run-p quality:lint:*",
     "quality:lint:packages": "lerna run --parallel quality:lint",
-    "quality:lint:root": "markdownlint README.md CHANGELOG.md docs/**/*.md",
+    "quality:lint:markdown": "markdownlint README.md CHANGELOG.md docs/**/*.md",
```

Comment:
The root `package.json` should trigger the actions of sub-packages and at the same time run actions for the root files (outside of subp-packages) if needed.

I would suggest the following:
```
    "quality:lint:packages:markdown": "lerna run --parallel quality:lint:markdown",
    "quality:lint:root": "run-p quality:lint:root:*",
    "quality:lint:root:markdown": "markdownlint README.md CHANGELOG.md docs/**/*.md",
    "quality:lint:root:prettier": "prettier --check **/*.{js,jsx,ts,tsx,css,scss,md,yaml,yml,json} !packages/**/* !demo/**/*",
```

Otherwise we will end up with files being linted and other not being linted. I didn't check the implementation myself, i just wrote it from memory :D.

Also I can handle this on the branch directly if you want.